config/client_config/dc/system/config/templates/features/aca_individual_market/aca_individual_market.yml

```diff
@@ -8,8 +8,10 @@ registry:
         item: :residency_attestation
         is_enabled: true
         settings:
-          - key: :effective_period
-            item: <%= Date.new(2020,6,6)..Date.new(2023,1,11) %>
+          - key: :effective_period_start
+            item: <%= ENV['RESIDENCY_SELF_ATTESTATION_EFFECTIVE_PERIOD_START'] || '' %>
```

Comment:
Is it safe to default these to empty? I think Date.parse('') throws an exception, just want to confirm that if the env isnt set, the logic doesnt fall over.

Reply:
Date.parse('') will indeed throw an exception.  My thinking here was that we can check for the presence of a value before parsing either of these in the self_attest_residency_enabled? method since all of the logic related to these config values is isolated there.  Admittedly, the same effect could be achieved without a fallback value (nil is returned anyways if the env var is missing).  Thinking about this now though, I think it makes sense to use the currently hardcoded values as the fallbacks.